### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by reschadahmadi**

### CODE REVIEW

File: app/assets/stylesheets/pages/_home.scss [hunk 1]
```diff
@@ -1 +1,21 @@
 // Specific CSS for your home-page
+.main-img{
```
Comment (i-need-coffee) [inline app/assets/stylesheets/pages/_home.scss]: same content as for the other one, you should not use this file, create a new file called "_welcome.scss" and repeat the steps written before

File: app/views/pages/welcome.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<section class="welcome-section">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-md-6 col-lg-4">
+        <h2 class="heading-section">Bienvenue sur Barathon</h2>
```
Comment (i-need-coffee) [inline app/views/pages/welcome.html.erb]: The app is in english, you need to translate the sentences, also the name app is not Barathon anymore, but BarFever : "Welcome to BarFever" "We know the perfect place"

**Review by i-need-coffee — CHANGES_REQUESTED** [review]
Again well done! A few changes and we're good to go!

File: app/assets/stylesheets/pages/_home.scss (hunk 1 above)
Comment (reschadahmadi) [reply app/assets/stylesheets/pages/_home.scss]: @ShadyaJ I fixed this and added sperate _welcome.scss

File: app/views/pages/welcome.html.erb (hunk 2 above)
Comment (reschadahmadi) [reply app/views/pages/welcome.html.erb]: @ShadyaJ I fixed this too. Thank you

**Review by i-need-coffee — APPROVED** [review]

**Pull request merged**